src/simplicial-hott/04a-jorthogonality.rzk.md

```diff
@@ -0,0 +1,126 @@
+# 4a. Right orthogonal fibrations
+
+This is a literate `rzk` file:
+
+```rzk
+#lang rzk-1
+```
+
+For every shape inclusion `ϕ ⊂ ψ`,
+we obtain a fibrancy condition for a map `α : A' → A`
+in terms of unique extension along `ϕ ⊂ ψ`.
+This is a relative version of unique extension along `ϕ ⊂ ψ`.
+
+We say that `α : A' → A` is _j-orthogonal_ to the shape inclusion `ϕ ⊂ ψ`,
+or a right orthogonal fibration with respect to `ϕ ⊂ ψ`,
+if the square
+```
+(ψ → A') → (ψ → A)
+
+   ↓          ↓
+
+(ϕ → A') → (ϕ → A)
+```
+is homotopy cartesian.
+
+```rzk title="BW23, Section 3"
+#def is-j-orthogonal-to-shape
+  ( I : CUBE)
+  ( ψ : I → TOPE )
+  ( ϕ : ψ → TOPE)
+  ( A' A : U)
```

Comment:
In this case there is no real reason to prefer A' A compared to A B. So I don't mind changing it.

In other places, where one also has multiple set of types, I think using primes is better, because it makes it clear how the variables are paired up. For example, a commutative square A' A C' C is much clearer than just A B C D.

Reply:
Another advantage when using primes, is that one can decorate simplices `\phi \to A` or `\phi \to A'` with matching primes to make it easier to see which simplices take values where.

Reply:
Yes, I see the benefit of using primes in some definitions. It's just here that I think it's not as useful :) Again, not insisting!

Reply:
After some deliberation I have decided to stick with `A'`, because having matching primes actually helped me a lot to know where the diagrams live.

Reply:
Good!